src/ucm/event/event.c

```diff
@@ -465,10 +465,8 @@ void ucm_event_handler_remove(ucm_event_handler_t *handler)
 
 static int ucm_events_to_native_events(int events)
 {
-    int native_events;
+    int native_events = events;
 
-    native_events = events & ~(UCM_EVENT_VM_MAPPED | UCM_EVENT_VM_UNMAPPED |
```

Comment:
why needed?

Reply:
because we need to save "installed events" including aggregate events, else test for virtual event is always fail.
UCM_EVENT_MEM_TYPE_ALLOC | UCM_EVENT_MEM_TYPE_FREE removed due to debug purposes

Reply:
maybe we should add aggregate events to ucm_mmap_installed_events then?
e.g pass the original events to ucm_mmap_install().
ucm_mmap_install() will convert the event flags to native events and try to install them.
then, in the code below, if the installation was successful, update ucm_mmap_installed_events with all events, including aggregate.

```
    /* status == UCS_OK */
    ucm_mmap_installed_events |= events;
    ucm_debug("mmap installed events = 0x%x", ucm_mmap_installed_events);
```

Reply:
ok, done